CMakeLists.txt

```diff
@@ -36,7 +34,12 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 set(warnings "-Wno-deprecated-declarations -Wall -Wextra -Werror -Wpedantic -Wno-implicit-fallthrough")
 if (APPLE)
     set(warnings "${warnings} -Wno-extended-offsetof")
-endif(APPLE)
+else()
+    # for beast and gcc release builds...
+    if("${CMAKE_BUILD_TYPE}" STREQUAL "Release")
+        set(warnings "${warnings} -Wno-maybe-uninitialized")
```

Comment:
Seems odd that we only need this for Release builds, is this perhaps catching something legitimate?

Reply:
I have been edified on this topic